app/src/main/java/com/example/habitsmasher/Habit.java

```diff
@@ -14,7 +15,7 @@
     private String _reason;
     private Date _date;
     private HabitEventList _habitEvents;
-    private String _days;
+    private ArrayList<String> _days;
```

Comment:
_days is now an arraylist of strings, doesn't change too much since you can just get the string of days by passing it through a DaysTracker object.